oss_src/sframe/sarray_file_format_v2.hpp

```diff
@@ -636,6 +636,48 @@ class sarray_group_format_writer_v2: public sarray_group_format_writer<T> {
   }
 
   /**
+   * Writes a collection of rows to a column
+   */
+  void write_column(size_t columnid, 
+                    size_t segmentid, 
+                    const std::vector<T>& t) {
+    DASSERT_LT(segmentid, m_nsegments);
+    DASSERT_LT(columnid, m_column_buffers.size());
+    DASSERT_EQ(m_array_open, true);
+
+    auto el_before_flush = m_column_buffers[columnid].elements_before_flush;
+    auto& buffer = m_column_buffers[columnid].segment_data[segmentid];
+    for(const auto& elem: t) {
+      buffer.push_back(elem);
+      if (buffer.size() >= el_before_flush) {
+        flush_block(columnid, segmentid);
+        el_before_flush = m_column_buffers[columnid].elements_before_flush;
```

Comment:
Yes. Though, I could just change el_before_flush to be a reference and I don't need this line.